tests/App/Resources/config/config.yaml

```diff
@@ -41,7 +41,7 @@ twig:
     exception_controller: null
     paths:
         - "%kernel.project_dir%/Resources/views"
-    strict_variables: "%kernel.debug%"
+    strict_variables: true
```

Comment:
we should probably test with this always enabled. cc @VincentLanglet

Reply:
I strongly agree. This should be done on every sonata project.
(I just did the same in test and even in prod in my work project).